test/test_crypto.py

```diff
@@ -208,3 +262,50 @@ def test_nv_getname(self):
         ename = self.ectx.TR_GetName(handle)
 
         self.assertEqual(ename.name, oname.name)
+
+    def test_public_from_pem_rsa_pem_cert(self):
+        pub = TPMT_PUBLIC()
+        crypto.public_from_pem(rsa_cert, pub)
```

Comment:
maybe refactor the function name to public_from_cert?

Reply:
I'll rename it to public_from_encoding so the name is generic